src/components/picker/picker.tsx

```diff
@@ -301,7 +310,16 @@ export class Picker {
         return null;
     }
 
-    private createChips(value: ListItem | ListItem[]): Chip[] {
+    private getValueId = (item: ListItem) => {
+        const value = item.value;
+        if (!!value && typeof value === 'object') {
+            return value.id;
+        }
+
+        return value;
```

Comment:
@adrianschmidt is there any way the `item.value = {}` if so do we need to handle the case somewhere?

Reply:
The types indicate that `value` must be a string, a number, or an object with a property `id`, which is a string or a number. So `value` should never be an empty object, or an object without an `id`. But if it is, then the same bug will still be there. I don't think we need to support that case though. We wouldn't really have had to support the case with an object that has an `id` either, since the types already specified what's valid and what's not. But since many will likely want to use these with limeobjects‚ I guess it's helpful to allow that.

Although, if they decide to use the picker with a mix of limeobjects from different limetypes, the bug could still appear, if two picked objects had the same `id`, but different `limetypes`.

Once we've merged the changes to chip-set, I think we might want to look that over. I don't think there's any particular reason we need to compare things as strings, but we couldn't change that here without changing the `Chip` interface.